### PR TITLE
fix: direct indirect links text

### PR DESCRIPTION
**Pull request by AdiGajbhiye**
## Checklist

- [ ] I have run this code and it appears to resolve the stated issue
- [ ] `README.md` updated and added information about my change

### CODE REVIEW

File: new_lineage_panel/src/App.tsx [hunk 1]
```diff
@@ -310,18 +310,18 @@ function App() {
               <div className="d-flex gap-sm">
                 <div className="d-flex gap-xxs align-items-center">
                   <DirectEdgeIcon />
-                  <div>Direct</div>
+                  <div>Select</div>
                   <InfoIcon
                     id="direct_lineage"
```
Comment (anandgupta42) [inline new_lineage_panel/src/App.tsx:315]: nit: change the id also?

File: new_lineage_panel/src/App.tsx [hunk 2]
```diff
@@ -310,18 +310,18 @@ function App() {
...
-                    message="Direct Linkages are shown if there is direct flow of data between columns through select statements."
+                    message="Select linkages are shown if there is direct flow of data between columns through select statements."
                   />
                 </div>
                 <div className="d-flex gap-xxs align-items-center">
                   <IndirectEdgeIcon />
-                  <div>Indirect</div>
+                  <div>Non-Select</div>
                   <InfoIcon
                     id="indirect_lineage"
```
Comment (anandgupta42) [inline new_lineage_panel/src/App.tsx:323]: nit: change the id also?

**Comment by mdesmet** [comment]
@AdiGajbhiye : Can you take care of the ids, this looks good to merge otherwise.

**Review by anandgupta42 — APPROVED** [review]
LGTM

**Pull request merged**